lib/restforce/db/client.rb

```diff
@@ -0,0 +1,52 @@
+module Restforce
+
+  module DB
+
+    # Restforce::DB::Client is a thin abstraction on top of the default
+    # Restforce::Data::Client class, which adds support for a handful of useful
```

Comment:
At the time I wrote the comment, I had also added the `getUpdated` endpoint, but found it less useful than our direct queries, so I removed it.